app.js

```diff
@@ -80,10 +80,13 @@ app.use('/components/jquery', express.static(path.join(__dirname, 'node_modules'
 app.use('/components/magnific-popup', express.static(path.join(__dirname, 'node_modules', 'magnific-popup', 'dist')));
 app.use('/components/scrollreveal', express.static(path.join(__dirname, 'node_modules', 'scrollreveal', 'dist')));
 
-app.use('/', require('./routes/index')(express, passport));
+if (config.util.getEnv('NODE_ENV') !== 'docker') {
```

Comment:
Can we have a comment, for exmaple,
// We do not open all WATT's features in docker/aws environment yet.

Reply:
done